src/noise_interfaces/noise_process_interface.jl

```diff
@@ -480,8 +480,17 @@ end
         push!(W.Z,copy(out2))
       end
     end
+    out1 .= W.curW + W.dW
```

Comment:
so lines 471 and 474 are incorrect and should be deleted?

Reply:
is it? I thought for `save_everystep` it should be there